mllm/backends/cpu/kernels/common/elewise-inl.hpp

```
@@ -64,31 +61,91 @@ struct DivOp {
-  __elementwise(x, y, n, DivOp{});
+HWY_NOINLINE HWY_MAYBE_UNUSED void elewise_div_fp32(mllm_fp32_t* out, const mllm_fp32_t* x, const mllm_fp32_t* y, size_t n) {
+  __elementwise(x, y, out, n, DivOp{});
 }
 
 //===----------------------------------------------------------------------===//
 // Elementwise + - * / By Const
 //===----------------------------------------------------------------------===//
 
-// TODO
+template<typename T, typename Op>
+HWY_INLINE void __elementwise_scalar(T* HWY_RESTRICT out, const T* HWY_RESTRICT x, const T y, size_t count, Op&& op) {
```

Clang (14.0.6)

[error] declaration uses identifier '__elementwise_scalar', which is a reserved identifier

(bugprone-reserved-identifier,-warnings-as-errors)